meetings/2024-01-17.md

```diff
@@ -0,0 +1,161 @@
+
+
+## Present
+* [Michiel de Jong](https://michielbdejong.com) (chair)
+* [Rahul Gupta](https://cxres.pages.dev/profile#i)
+* Maxime Lecoq
+* Hadrian Zbarcea
+* [Sarven Capadisli](https://csarven.ca/#i)
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* [Virginia Balseiro](https://virginiabalseiro.com/#me)
+* Reza Soltani
+* [April Daly](https://aprildaly.me)
+* [Pierre-Antoine Champin](https://solid.champin.net/pa/profile/card#me)
+* michal/mrkvon
+* Fred Gibson
+
+---
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Sarven Capadisli
+* Virginia Balseiro
+
+---
+
+## Topics
+
+### Open spec PRs
+
+#### Add security consideration for serving user-created files
+URL: https://github.com/solid/specification/pull/598
+
+* VB: Please review.
+* SC: Tied to an issue, some discussion in the PR.Wrote a comment earlier. Thought this through for a while. I think the consideration is good and we should have more stuff along those lines. Depending on whether the protocol expresses a need like that says something else about the architecture. There's some assumptions baked into this. It has some tradeoffs/conflicts with other things. We should weigh a bit more and discuss further before assuming it is as intended. Does this belong in that general pattern? Is it consistent with the rest of the design? Just my considerations.
+* MdJ: Makes sense. 
+* eP: We should add it as soon as possible in whatever form we reach rough consensus. Still at least list the possible issue. We can't expect everyone to be aware of the issues. It could be an inline issue to make people aware possible exists. And perhaps people give valuable feedback.
```

Comment:
```suggestion
* eP: We should add it as soon as possible in whatever form we reach rough consensus. Still at least list the possible issues. We can't expect everyone to be aware of all the issues. It could be an inline issue to help make people aware it exists. And perhaps people give valuable feedback.
```